metricbeat/module/munin/node/_meta/data.json

```diff
@@ -0,0 +1,207 @@
+                "rxdrop": 0,
+                "trans": 0,
+                "txdrop": 0
+            },
+            "if_eth0": {
+                "down": 29044,
+                "up": 29797
+            },
+            "if_ip6tnl0": {
+                "down": 0,
+                "up": 0
+            },
+            "if_tunl0": {
+                "down": 0,
+                "up": 0
+            },
+            "interrupts": {
+                "ctx": 2170557862,
+                "intr": 1118035320
+            },
+            "irqstats": {
+                "i0": 29,
+                "i24": 0,
+                "i25": 2163,
+                "i26": 5676056,
+                "i27": 2655,
+                "i28": 52,
+                "i29": 2005,
+                "i30": 0,
+                "i31": 1633298,
+                "i32": 1,
+                "i33": 0,
+                "i34": 18902840,
+                "i35": 24222591,
+                "i36": 0,
+                "i4": 668,
+                "i8": 1,
+                "i9": 0,
+                "iCAL": 127014653,
+                "iERR": 0,
+                "iIWI": 0,
+                "iLOC": 445614790,
+                "iMIS": 0,
+                "iNMI": 0,
+                "iPIN": 0,
+                "iPIW": 0,
+                "iPMI": 0,
+                "iRES": 494963671,
+                "iRTR": 0,
+                "iSPU": 0,
+                "iTLB": 113668705
+            },
+            "load": {
+                "load": 0.16
+            },
+            "memory": {
+                "active": 1087918080,
+                "apps": 328413184,
+                "buffers": 493154304,
+                "cached": 1458929664,
+                "committed": 958160896,
+                "free": 2490265600,
+                "inactive": 1149038592,
+                "mapped": 77598720,
+                "page_tables": 2609152,
+                "slab": 415215616,
+                "swap": 22421504,
+                "swap_cache": 1196032,
+                "vmalloc_used": 0
+            },
+            "open_files": {
+                "max": 524288,
+                "used": 576
+            },
+            "open_inodes": {
+                "max": 119134,
+                "used": 56297
+            },
+            "proc_pri": {
+                "high": 0,
+                "locked": 0,
+                "low": 0
+            },
+            "processes": {
+                "dead": 0,
+                "paging": 0,
+                "processes": 6,
+                "runnable": 2,
+                "sleeping": 4,
+                "stopped": 0,
+                "uninterruptible": 0,
+                "zombie": 0
+            },
+            "swap": {
+                "swap_in": 569,
+                "swap_out": 7814
+            },
+            "threads": {
+                "threads": 6
+            },
+            "uptime": {
+                "uptime": 5.98
+            },
+            "users": {
+                "X": 0,
```

Comment:
Maybe we should open an issue to lowercase this field... or rename it to something more meaningful (well... maybe it's super meaningful for people experienced in Munin)

Reply:
> maybe it's super meaningful for people experienced in Munin

:laughing: